src/analysis/retail/evoker/preservation/modules/talents/EssenceBurst.tsx

```diff
@@ -11,17 +11,27 @@ import { SPELL_COLORS } from 'analysis/retail/evoker/preservation/constants';
 import DonutChart from 'parser/ui/DonutChart';
 import { SpellLink } from 'interface';
 import { t } from '@lingui/macro';
+import ItemManaGained from 'parser/ui/ItemManaGained';
 
 const ESSENCE_COSTS: { [name: string]: number } = {
   'Emerald Blossom': 3,
   Echo: 2,
   Disintegrate: 3,
 };
 
+const MANA_COSTS: { [name: string]: number } = {
+  'Emerald Blossom': 0.048,
+  Echo: 0.017,
+  Disintegrate: 0,
+};
+
+const TOTAL_MANA = 250000;
```

Comment:
Verify that the mana returns are based off Base Mana and not Enchanted mana

Reply:
Was just using this to caclulate manaCost, shouldn't be necessary when I switch to using spellbook mana cost.